docs/kusion/2-getting-started/2-deliver-wordpress.md

```diff
@@ -0,0 +1,208 @@
+```
+
+You can replace the `runtimes.kubernetes.kubeConfig` field with your own kubeconfig file path in `workspace.yaml` and execute the following command line to initiate the workspace configuration for `dev` stack. 
+
+```shell
+kusion workspace create dev -f workspace.yaml
+```
+
+You can use the following command lines to list and show the workspace configurations for `dev` stack. 
+
+```shell
+kusion workspace list
+
+kusion workspace show dev
+```
+
+The `workspace.yaml` is a sample configuration file for workspace management, including `Kubernetes` runtime config and `MySQL` module config. Workspace configurations are usually declared by **Platform Engineers** and will take effect through the corresponding stack. 
+
+:::info
+More details about the configuration of Workspace can be found in [Workspace Management](https://github.com/KusionStack/kusion/blob/main/docs/design/workspace_management/workspace_management.md). 
+:::
+
+## Init Project
+
+We can start by initializing this tutorial project with online templates: 
+
+```shell
+kusion init --online
+```
+
+All init templates are listed as follows: 
+
+```shell
+➜  kusion_playground kusion init --online
+? Please choose a template: wordpress-local-db         A sample wordpress project with local database
+This command will walk you through creating a new kusion project.
+
+Enter a value or leave blank to accept the (default), and press <ENTER>.
+Press ^C at any time to quit.
+
+Project Config:
+? ProjectName: wordpress-local-db
+? AppName: wordpress
+Stack Config: dev
+? Image: wordpress:6.3
+Created project 'wordpress-local-db'
+```
+
+Please select `wordpress-local-db` and press `Enter`, after which we will see the hints below and use the default values to configure this project and stack. 
+
+![](/img/docs/user_docs/getting-started/init-wordpress-local-db.gif)
+
+The directory structure looks like the following: 
+
+```shell
+cd wordpress-local-db/dev && tree
+```
+
+```shell
+➜  kusion_playground cd wordpress-local-db/dev && tree
+.
+├── kcl.mod
+├── main.k
+└── stack.yaml
+
+1 directory, 3 files
+```
+
+:::info
+More details about the directory structure can be found in [Project](../3-concepts/1-project/1-overview.md) and [Stack](../3-concepts/2-stack/1-overview.md). 
+:::
+
+### Review Configuration Files
+
+Now let's have a glance at the configuration file of `dev/main.k`: 
+
+```python
+import catalog.models.schema.v1 as ac
+import catalog.models.schema.v1.workload as wl
+import catalog.models.schema.v1.workload.container as c
+import catalog.models.schema.v1.workload.network as n
+import catalog.models.schema.v1.accessories.mysql
+
+# main.k declares customized configurations for dev stack.
+wordpress: ac.AppConfiguration {
+    workload: wl.Service {
+        containers: {
+            wordpress: c.Container {
+                image: "wordpress:6.3"
+                env: {
+                    "WORDPRESS_DB_HOST": "$(KUSION_DB_HOST_WORDPRESS_MYSQL)"
```

Comment:
Please add comments or a reference link to explain these magic values.

Reply:
The explanation of these magic values has been added below already: 

> The necessary Kubernetes resources for deploying and using the local database will be generated, and users can get the `host`, `username` and `paasword` of the database through the [mysql credentials and connectivity](../6-reference/2-modules/1-catalog-models/database/mysql.md#credentials-and-connectivity) of Kusion in application containers.